CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+
+- Attempting to create a `Uri` object from a malformed URI will now throw a `MalformedURIException`,
+  not `InvalidArgumentException`. Callers realying on the exception being thrown to detect invalid
```

Comment:
this is misleading. an InvalidArgumentException is still thrown as MalformedURIException extends InvalidArgumentException

Reply:
Reworded